sea-query-driver/src/sqlx_mysql.rs

```diff
@@ -106,24 +106,26 @@ pub fn bind_params_sqlx_mysql_impl(input: TokenStream) -> TokenStream {
     output.into()
 }
 
-pub fn sea_query_driver_mysql_impl() -> TokenStream {
+pub fn sea_query_driver_mysql_impl(input: TokenStream) -> TokenStream {
+    let sqlx_path = parse_macro_input!(input as DriverArgs);
+
     let output = quote! {
         mod sea_query_driver_mysql {
-            use sqlx::{mysql::MySqlArguments, MySql};
-            use sea_query::{Value, Values};
+            use #sqlx_path::sqlx::{mysql::MySqlArguments, MySql};
+            use ::sea_query::{Value, Values};
```

Comment:
Would this still work without prefixing :: on sea_query?
I think it was made without :: because we can potentially use a re-exported sea-query in scope

Reply:
We create a new scope (mod). if we support the re-export sea-query need a new parameter for macro.

Reply:
@tyt2y3 I added support for `::sea-query` prefix.